### PR TITLE
(#395) - Allow DOS line-breaks

### PR DESCRIPTION
**Pull request by andreoss**
Per #395:
- Convert DOS line breaks to UNIX format before parsing

### CODE REVIEW

File: eo-parser/src/main/java/org/eolang/parser/Syntax.java [hunk 1]
```diff
@@ -50,6 +56,11 @@
  */
 public final class Syntax {
 
+    /**
+     * Line break.
+     */
+    private static final String NEW_LINE = "\n";
```
Comment (yegor256) [inline eo-parser/src/main/java/org/eolang/parser/Syntax.java:62]: @andreoss what's the point of making it a const? You use it only once. The second usage can be removed. Check this out https://www.yegor256.com/2015/07/06/public-static-literals.html

**Comment by yegor256** [comment]
@andreoss see one comment above

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` is banned at [#441](https://github.com/cqfn/eo/issues/441); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

File: eo-parser/src/main/java/org/eolang/parser/Syntax.java (hunk 1 above)
Comment (andreoss) [reply eo-parser/src/main/java/org/eolang/parser/Syntax.java]: @yegor256 Removed

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@andreoss thanks!

**Pull request merged**